src/common/gpx.c

```diff
@@ -20,6 +20,10 @@
 #include "common/darktable.h"
 #include <glib.h>
 #include <inttypes.h>
+#include <math.h>
+
+#define PI 3.1415926535
```

Comment:
you don't have to redefine it, we have a `M_PI_F` constant already defined in https://github.com/darktable-org/darktable/blob/eb70a243e61f814f6b198c244e0096aa5265f46f/src/common/math.h#L40

Reply:
ok! but is `#include <math.h>` needed?

Reply:
Not if you directly `#include "common/math.h"`, it's included from there too.

Reply:
I haven't undestood... must I change `#include <math.h>` to `#include "common/math.h"`?

Reply:
Sorry, yes.

Reply:
done, thank you! :-)